stl/inc/ostream

```diff
@@ -912,13 +912,39 @@ basic_ostream<char, _Traits>& operator<<(
 }
 
 #ifdef __cpp_char8_t // These deleted overloads are specified in P1423.

```

Comment:
DECISION REQUIRED: The non-`char8_t` overloads should not be `#ifdef __cpp_char8_t`. They should be either unconditional - which I believe is SG16's intent - or `_HAS_CXX20` if we're happy to continue allowing e.g. `cout << char32_t{/*...*/}` to output an `unsigned int`.

I'm inclined not to break C++17 programs. Thoughts?

Reply:
Our usual rule is that if it's source-breaking, we guard it for C++20 - that's the whole reason we have Standard modes. We had to do that fairly recently for LWG-2221 "No formatted output operator for `nullptr`", where we guarded it for C++17 due to its source-breaking impact on C++14.

Reply:
Yes, this is a similar case: WG21 has made a statement that this functionality was always broken and unintended, and that it should be disabled unconditionally. I think that we should address it similarly as well, by vetoing applying this breaking change to sub-C++20 modes. I just want to be sure that we're making that choice deliberately and with full information.

Reply:
Can we make them work but be `[[deprecated]]` in 17 and earlier modes?

Reply:
Hm, I guarded the deleted overloads with `_HAS_CXX20`. 

As far as I know if I would add a deprecated specialization of those overloads for pre C++20 I would have to implement them. I would rather not do that?!

Reply:
We discussed this a bit in our weekly STL sync, and there's consensus for the current approach (Guard with `_HAS_CXX20`, don't mess with implementing new overloads so we can tag them `[[deprecated]]`.